MetaMask/metamask-extension
# Update content for liquidity source info

## Description
Updates content for liquidity source info in a tooltip.

## Testing
- In Swaps request quotes
- Open a quotes list and then click on an individual quote to see quote detail
- Check the tooltip next to `Liquidity source`
